types/firestore.d.ts

```diff
@@ -2090,7 +2089,6 @@ declare namespace FirebaseFirestore {
    */
   export type AggregateType = 'count' | 'avg' | 'sum';
```

Comment:
I believe we also need to remove `@internal` from this type so that the type is exposed in documentation.